.circleci/config.yml

```diff
@@ -1,32 +0,0 @@
-# Javascript Node CircleCI 2.0 configuration file
```

Comment:
The `.circleci` directory reference in `.npmignore` can now be removed (and replaced by a `.github` entry):
https://github.com/forcedotcom/sf-fx-sdk-nodejs/blob/1d10c6a0cdc3ec76a9e697b1fc6079ee167bbca9/.npmignore#L2

Reply:
i'm going to leave this as is for a couple of reasons:
- `.npmignore` shouldn't even be necessary since the `files` field is used in `package.json` so there should be no worries about publishing the `.github` folder
- we'll no longer be publishing any artifacts from this repository once #146 is merged